addons/point_of_sale/static/src/app/components/order_tabs/order_tabs.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates id="template" xml:space="preserve">
     <t t-name="point_of_sale.OrderTabs">
-        <ListContainer items="orders"
+        <ListContainer
+            items="orders"
             onClickPlus="() => this.newFloatingOrder()"
             t-slot-scope="scope"
             class="props.class"
-            forceSmall="ui.isSmall and !env.inDialog"
+            show="showOderTabs()"
```

Comment:
i don't really get this? If the goal is to hide the whole thing, why not use `t-if`? Also, how will you now add new floating orders in mobile ?

Reply:
indeed, it should be t-if.